linkerd/app/outbound/src/tcp/logical.rs

```diff
@@ -90,9 +86,10 @@ where
             .push(svc::UnwrapOr::layer(
                 endpoint
                     .clone()
-                    .push_map_target(Endpoint::from_logical(
-                        tls::NoClientTls::NotProvidedByServiceDiscovery,
-                    ))
+                    .push_map_target(|logical: Logical| {
+                        debug!("No profile resolved");
```

Comment:
nit: should this include the logical endpoint we didn't resolve a profile for?

Reply:
the address is already in a parent span